AssertionError: 540.0 != 2

### Information about bug

======================================================================
FAIL: test_bundle_balance_report_T_PBB_001 (erpnext.stock.report.product_bundle_balance.test_product_bundle_balance.TestProductBundleBalance)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/ubuntu/frappe-bench/apps/erpnext/erpnext/stock/report/product_bundle_balance/test_product_bundle_balance.py", line 70, in test_bundle_balance_report_T_PBB_001
    self.assertEqual(parent_row["bundle_qty"], 2)
AssertionError: 540.0 != 2

----------------------------------------------------------------------
Ran 1 test in 1.761s

FAILED (failures=1)

### Module

stock

### Version

Assets: v0.0.1 (pre-prod)

ERPNext: v15.59.0 (pre-prod)

ERPNext CRM: v0.0.1 (pre-prod)

ERPNext India: v0.0.1 (pre-prod)

Frappe Framework: v15.67.0 (pre-prod)

India Compliance: v15.17.1 (pre-prod)

Payments: v0.0.1 (version-15)

Projects: v0.0.1 (pre-prod)

Sales Commission: v0.0.1 (pre-prod)

### Installation method

None

### Relevant log output / Stack trace / Full Error Message.

```shell

```

--- a/erpnext/stock/report/product_bundle_balance/test_product_bundle_balance.py
+++ b/erpnext/stock/report/product_bundle_balance/test_product_bundle_balance.py
@@ -4,9 +4,6 @@ from frappe.utils import nowdate
 
 from erpnext.selling.doctype.product_bundle.test_product_bundle import make_product_bundle
 from erpnext.stock.doctype.item.test_item import create_item
-
-# from erpnext.accounts.doctype.company.test_company import create_company
-# from erpnext.stock.doctype.product_bundle.product_bundle import make_product_bundle
 from erpnext.stock.doctype.stock_entry.test_stock_entry import make_stock_entry
 from erpnext.stock.doctype.warehouse.test_warehouse import create_warehouse
 from erpnext.stock.report.product_bundle_balance.product_bundle_balance import execute
@@ -28,10 +25,10 @@ class TestProductBundleBalance(FrappeTestCase):
 
 		# Parent (bundle) item
 		self.parent_item = create_item(
-			item_code="_Test Product Bundle Item New", is_stock_item=0, company=self.company
+			item_code="_Test Product Bundle Item New -Test", is_stock_item=0, company=self.company
 		)
 
-		make_product_bundle("_Test Product Bundle Item New", ["BUNDLE-CHILD-1", "BUNDLE-CHILD-2"], 2)
+		make_product_bundle("_Test Product Bundle Item New -Test", ["BUNDLE-CHILD-1", "BUNDLE-CHILD-2"], 2)
 
 		# Add stock for child items: 5 units of child 1, 9 units of child 2
 		make_stock_entry(

